examples/resource_partitioner/async_customization.cpp

```diff
@@ -502,27 +502,33 @@ int main(int argc, char** argv)
         hpx::threads::policies::example::shared_priority_queue_scheduler<>;
     using hpx::threads::policies::scheduler_mode;
     // setup the default pool with our custom priority scheduler
-    rp.create_thread_pool(
-        "custom",
-        [](hpx::threads::policies::callback_notifier& notifier, std::size_t num_threads,
-          std::size_t thread_offset, std::size_t pool_index,
-          std::string const& pool_name) -> std::unique_ptr<hpx::threads::thread_pool_base>
-          {
+    rp.create_thread_pool("custom",
+        [](hpx::threads::policies::callback_notifier& notifier,
+            std::size_t num_threads, std::size_t thread_offset,
+            std::size_t pool_index, std::string const& pool_name,
+            hpx::threads::detail::network_background_callback_type const&
+                network_background_callback,
+            hpx::threads::policies::detail::affinity_data const& affinity_data,
+            std::size_t max_background_threads, std::size_t max_idle_loop_count,
+            std::size_t max_busy_loop_count)
```

Comment:
May I suggest to combine all of the `size_t`'s in a special struct to reduce the number of arguments?

Reply:
Yep, will do, it's getting a bit out of hand.